docs/reference/indices/diskusage.asciidoc

```diff
@@ -0,0 +1,160 @@
+[[indices-disk-usage]]
+=== Analyze index disk usage API
+++++
+<titleabbrev>Analyze index disk usage</titleabbrev>
+++++
+
+experimental[]
+
+Analyzes the disk usage of each field of an index. This API
+might not support indices created in previous {es} versions.
+
+[source,console]
+--------------------------------------------------
+POST /my-index-000001/_index_disk_usage?run_expensive_tasks=true
+--------------------------------------------------
+// TEST[setup:my_index]
+
+
+[[analyze-index-disk-usage-api-request]]
+==== {api-request-title}
+
+`POST /<index>/_index_disk_usage`
+
+[[analyze-index-disk-usage-api-request-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage`
+<<privileges-list-indices,index privilege>> for the target index or index alias.
+
+
+[[analyze-index-disk-usage-api-path-params]]
+==== {api-path-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index]
++
+By default, you must explicitly name the indices you are analyzing the disk usage.
+To specify indices to close with `_all`, `*`, or other wildcard
+expressions, change the `action.destructive_requires_name` setting to `false`.
+You can update this setting in the `elasticsearch.yml` file
+or using the <<cluster-update-settings,cluster update settings>> API.
```

Comment:
I added this because I prefer this API to accept a single API. Should we still enforce this, but not a destructive action? If so then we can drop `_index` from `_index_disk_usage` API?